app/controller/SettingsController.js

```diff
@@ -115,7 +115,20 @@ SDL.SettingsController = Em.Object.create(
     getStatusUpdate: function() {
       FFW.BasicCommunication.GetStatusUpdate();
     },
+    stopStreams: function() {
+      var appID = SDL.SDLController.model.appID;
+      SDL.StreamAudio.stop();
+      if(SDL.SDLController.getApplicationModel(appID).audioStreamingStarted) {
+        SDL.InfoController.stopStreamingAdapter('audio').then(function() {
+          Em.Logger.log('Audio playback stopped');
+        })
+        .catch(error => {
+          Em.Logger.log('Stop audio streaming adapter failed');
+        });
+      }
+    },
     phoneCall: function() {
+      this.stopStreams();
```

Comment:
@LiliiaShlikhtLuxoft Did you try smthg like in the suggestion below ?
```suggestion
    phoneCall: function() {
      SDL.SDLModel.stopAudioStream();
```

Reply:
@ValeriiMalkov done! Please check one more time